src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -61,3 +63,19 @@ if (shouldAutoStart()) {
     }
   });
 }
+
+async function intiializeConfigAsync(platform: Platform, resourceLoader: WebAssemblyResourceLoader) : Promise<void> {
+  const configFiles = resourceLoader.readConfigFilesAsync();
+  const resolvedFiles = await Promise.all(configFiles.map(async c => {
+    const content = new Uint8Array(await c.contentPromise);
+    return { name: c.name, content: content};
```

Comment:
Nit: I think the following would also work, though it's your stylistic choice:

```suggestion
    return { c.name, content };
```